lib/decoration-management.js

```diff
@@ -201,7 +201,7 @@ export default class DecorationManagement {
       }
     }
 
-    return decorationsByMarkerId
+    return Object.values(decorationsByMarkerId)
```

Comment:
Was this actually an array? I have not changed this.

Reply:
according to the jsdoc

```js
/**
   * Returns all the decorations that intersect the passed-in row range.
   *
   * @param  {number} startScreenRow the first row of the range
   * @param  {number} endScreenRow the last row of the range
   * @return {Array<Decoration>} the decorations that intersect the passed-in
   *                             range
   */
```

Reply:
We should either change the jsdoc or the return value

Reply:
I think here the dosc should change. Because the code shows that the intention has been to return an object. Maybe if we setup CI for running the tests for plugins, we can see if anyone is using this API.

Reply:
According to https://github.com/atom/atom/blob/v1.54.0/src/text-editor.js#L2664 it should be an object not an array. So we should change the docs.

Reply:
Fixed in https://github.com/atom-minimap/minimap/commit/0b8bf8b78c8a0a2f95c45feaaab27737d0b3353c